### PR TITLE
Get status and title from chromecast

### PR DESCRIPTION
**Pull request by jarlebh**
This pull request add support for getting the current app and title from the chromecast.
Tested with Spotify, NRK and Altibox.

### CODE REVIEW

**Comment by kaikreuzer** [comment]
Please ignore the Jenkins error...

**Comment by kaikreuzer** [comment]
Please forgive me closing and re-opening this PR, but I am just using it to get the Jenkins PR build working again.

File: addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/handler/ChromecastHandler.java [hunk 1]
```diff
@@ -326,6 +359,33 @@ private void handleCcMediaStatus(final MediaStatus mediaStatus) {
             default:
                 break;
         }
+        if (mediaStatus.media != null && mediaStatus.media.metadata != null) {
+            updateMediaStatus(mediaStatus.media);
+        }
+    }
+
+    private void updateMediaStatus(Media media) {
+        Object title = media != null ? media.metadata.getOrDefault("title", "") : "";
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/handler/ChromecastHandler.java]: Can these be strings or are they some strange type?

Comment (martinvw) [inline addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/handler/ChromecastHandler.java]: Should they be UNDEF when media is present?

File: addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/handler/ChromecastHandler.java [hunk 2]
```diff
@@ -326,6 +359,33 @@ private void handleCcMediaStatus(final MediaStatus mediaStatus) {
...
+        updateState(new ChannelUID(getThing().getUID(), ChromecastBindingConstants.CHANNEL_TITLE),
+                new StringType(title.toString()));
+        updateState(new ChannelUID(getThing().getUID(), ChromecastBindingConstants.CHANNEL_ALBUM),
+                new StringType(albumName.toString()));
+        updateState(new ChannelUID(getThing().getUID(), ChromecastBindingConstants.CHANNEL_ARTIST),
+                new StringType(artist.toString()));
+    }
+
+    private void handleApp(Application app) {
+        String name = "";
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/handler/ChromecastHandler.java]: Should they be UNDEF when media is present?

**Comment by martinvw** [comment]
O, and you should fix the merge conflicts and test it again after that, do you have other people testing as well?

**Comment by holmes** [comment]
I've created a similar but a bit more complete PR with regard to media metadata updates: https://github.com/openhab/openhab2-addons/pull/2097

**Comment by martinvw** [comment]
@jarlebh can you check whether the referred binding also implements the same or more as you did? If so I would prefer the pr which adds the most value, agree?

**Comment by martinvw** [comment]
@jarlebh can we close this PR in favor of the other one?

**Comment by martinvw** [comment]
Given that the other pr has been merged I believe this one should be closed if that is not correct please feel free to rebase and reopen

**Pull request closed without merging**